srcpkgs/ruby-build/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20211227
+version=20221121
```

Comment:
20221123 is available

Reply:
a new commit has been force pushed that updates the version